presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -8,6 +8,18 @@ HTTP/HTTPS, using the corresponding port number specified in the coordinator
 :ref:`config_properties`. It can be configured with :ref:`specific related
 properties <web-ui-properties>`.
 
+You can disable the Web UI, if you do not want to expose the information from
+the Web UI to your users.
```

Comment:
maybe reference the property.

Also maybe something like:
```
The Web UI can be completely disabled if desired by setting the ``web-ui.enabled`` configuration property.
```

Reply:
Added to my sentence.